x-pack/plugins/session_view/public/components/process_tree/helpers.ts

```diff
@@ -55,9 +54,9 @@ export const updateProcessMap = (processMap: ProcessMap, events: ProcessEvent[])
       processMap[id] = process;
     }
 
-    if (event.event?.kind === EventKind.signal) {
+    if (event.kibana?.alert) {
       process.addAlert(event);
-    } else if (event.event?.kind === EventKind.event) {
+    } else {
       process.addEvent(event);
```

Comment:
it can happen to add an empty event on `process.addEvent(event)`?

Reply:
yea, i made this change assuming data will always be one or the other, probably not a good assumption, i'll revert this change.